sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/models/EventHubNamespace.java

```diff
@@ -89,6 +89,16 @@ public interface EventHubNamespace extends
      */
     PagedIterable<EventHubNamespaceAuthorizationRule> listAuthorizationRules();
 
+    /**
+     * @return minimum tls version of the namespace
+     */
+    TlsVersion minimumTlsVersion();
+
+    /**
+     * @return zone redundant of the namespace
+     */
+    Boolean zoneRedundant();
```

Comment:
Does `null` mean zoneRedundant or not?

We'd expect to return `boolean`.

Reply:
The default value of `zoneRedundant` is false, so the return value will not be null.

Reply:
Then change return type to `boolean`.

Reply:
> Also use full Javadoc, not only `@return`. Add a description.

Fixed in the new version.